packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/StakeExchangeForm/StakeExchangeForm.tsx

```diff
@@ -203,6 +203,7 @@ const StakeExchangeForm = ({
                 selectedAddressValue: activeAccountAddress,
               }),
           }}
+          noOptionsMessage={() => 'No available Metamask address'}
```

Comment:
why is this a function?

Reply:
because `noOptionsMessage` prop is defined as a function by `react-select` library
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/9713b935-8595-4d33-81cc-89a0b5d40d77) ([source](https://react-select.com/props))